bootstrap-salt.sh

```diff
@@ -4538,6 +4542,13 @@ install_amazon_linux_ami_git_deps() {
         yum -y install ca-certificates || return 1
     fi
 
+    PIP_EXE='pip'
+    if [ "$(echo "$GIT_REV" | egrep -o '2[0-9]{3}')" -ge 2016 ]; then
```

Comment:
Thanks for this suggestion adn catching this I like your approach much better. I added in my latest commit.